css/src/components/checkbox.scss

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-color: $text !default;
```

Comment:
nit for clarity: `$checkbox-unchecked-border-color`

Reply:
updated